File: Selenium/Activities/Activity6.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.keys import Keys

driver = webdriver.Firefox()
actions = webdriver.ActionChains(driver)

driver.get("https://v1.training-support.net/selenium/input-events")

print(driver.title)

actions.key_down(Keys.SHIFT).send_keys("r").key_up(Keys.SHIFT).perform()
actions.key_down(Keys.CONTROL).send_keys("a").send_keys("c").key_up(Keys.CONTROL).perform()

driver.quit()

File: TestNG/Activities/Activity6.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Parameters;
import org.testng.annotations.Test;

public class Activity6 {

    WebDriver driver;

    @BeforeClass
    public void setup(){
        driver = new FirefoxDriver();
        driver.get("https://v1.training-support.net/selenium/login-form");
    }
    @AfterClass
    public void close(){
        driver.close();
    }
    @Test
    @Parameters({"username", "password"})
    public void test1(String username, String password) {
        driver.findElement(By.xpath("//*[@id=\"username\"]")).sendKeys(username);
        driver.findElement(By.xpath("//*[@id=\"password\"]")).sendKeys(password);
        driver.findElement(By.xpath("//*[@id=\"dynamic-attributes-form\"]/div/button")).click();
        System.out.println(driver.findElement(By.xpath("//*[@id=\"action-confirmation\"]")).getText());
    }

}
